fintech-examples/fintech-impl/src/main/java/de/adorsys/opba/fintech/impl/database/entities/SessionEntity.java

```diff
@@ -77,16 +65,10 @@ public static void validateSessionCookieValue(String sessionCookieValueString, S
         ObjectMapper mapper = new ObjectMapper();
         SessionCookieValue sessionCookieValue = mapper.readValue(decode, SessionCookieValue.class);
         if (sessionCookieValue.getHashedXsrfToken() == xsrfToken.hashCode()) {
-            log.info("validation of token ok");
+            log.info("validation of token for session ok {}", sessionCookieValue);
```

Comment:
While is out of the scope of this PR, hashCode() is not portable across JVMs. Probably using some secure hashes like SHA-2 with low collision probability and portability accross JVMs is better

Reply:
yep, use now sha256